### PR TITLE
Technical Review: Kubernetes

### PR DESCRIPTION
**Pull request by cbbayburt**
Minor fixes for Kubernetes doc page as part of https://github.com/SUSE/spacewalk/issues/9295

### CODE REVIEW

**Comment by cbbayburt** [comment]
@Loquacity, @jcayouette I left a couple of `TODO` comments where I need some input from you. Feel free to comment your feedback inline.

File: modules/administration/pages/kubernetes.adoc [hunk 1]
```diff
@@ -8,10 +8,11 @@
 
 The prerequisites listed below should be met before proceeding.
 
-* At least one _Kubernetes_ or _SUSE CaaS Platform _ cluster available on your network
+* At least one _Kubernetes_ or _SUSE CaaS Platform_ cluster available on your network
```
Comment (Loquacity) [inline modules/administration/pages/kubernetes.adoc]:
```suggestion
* At least one Kubernetes or SUSE CaaS Platform cluster available on your network
```

We can remove this markup, it's not required.

File: modules/administration/pages/kubernetes.adoc [hunk 2]
```diff
@@ -8,10 +8,11 @@
...
 * {productname} configured for container management
 +
 
+// TODO: Is "note" the right choice here?
 [NOTE]
 ====
 Required channels are present, a registered container build host available, etc.
```
Comment (Loquacity) [inline modules/administration/pages/kubernetes.adoc]: This isn't a full sentence. I have no idea what it's trying to say.

File: modules/administration/pages/kubernetes.adoc [hunk 3]
```diff
@@ -27,9 +28,10 @@ Required channels are present, a registered container build host available, etc.
 
 
 * Kubernetes version 1.5.0 or higher.
-Alternatively use SUSE CaaS Platform _(SUSE CaaS Platform includes Kubernetes 1.5.0 by default)_
+Alternatively use SUSE CaaS Platform
```
Comment (Loquacity) [inline modules/administration/pages/kubernetes.adoc]:
```suggestion
Alternatively, you can use SUSE CaaS Platform
```

File: modules/administration/pages/kubernetes.adoc [hunk 4]
```diff
@@ -27,9 +28,10 @@ Required channels are present, a registered container build host available, etc.
...
 * Docker version 1.12 or higher on the container build host
 
+// TODO: This is already mentioned under "Prerequisites". Should we keep it?
```
Comment (Loquacity) [inline modules/administration/pages/kubernetes.adoc]: No, you can just xref to it.

File: modules/administration/pages/kubernetes.adoc [hunk 5]
```diff
@@ -133,7 +135,7 @@ Under the `Overview` tab, notice the data in `Runtime` and `Instances` fields un
 ** Pod name
 ** Namespace which the pod resides in
 ** The runtime status of the container in the specific pod.
-For more about status icons, see the next section.
+For more about status icons, see <<at.obtain.runtime.data.about.images,Obtain Runtime Data about Images>>.
```
Comment (Loquacity) [inline modules/administration/pages/kubernetes.adoc]:
```suggestion
```

This is unnecessary.

File: modules/administration/pages/kubernetes.adoc (hunk 2 above)
Comment (cbbayburt) [reply modules/administration/pages/kubernetes.adoc]: It's an explanation of the line above: "{productname} configured for container management"

Sorry, my TODO line makes it less obvious I guess

Comment (Loquacity) [reply modules/administration/pages/kubernetes.adoc]: OK, so are you explaining what "configured for container management" means? If that is the case, I would put that explanation in the dot point, not in an admonition.

**Comment by cbbayburt** [comment]
@Loquacity ready for a final review now

**Pull request merged**